mmv1/third_party/terraform/services/container/go/node_config.go.tmpl

```diff
@@ -1362,6 +1384,8 @@ func flattenNodeConfigDefaults(c *container.NodeConfigDefaults) []map[string]int
 
 	result[0]["containerd_config"] = flattenContainerdConfig(c.ContainerdConfig)
 
+	result[0]["insecure_kubelet_readonly_port_enabled"] = c.NodeKubeletConfig.InsecureKubeletReadonlyPortEnabled
```

Comment:
Is `c.NodeKubeletConfig == nil` checked here? I saw this error in a build log from the latest run:

```
github.com/hashicorp/terraform-provider-google-beta/google-beta/services/container.flattenNodeConfigDefaults(0xc01eea6ea0)
	/go/src/github.com/modular-magician/terraform-provider-google-beta/google-beta/services/container/node_config.go:1367 +0xea
```

Reply:
Yes, see other comment, and was looking at this / fixing now. I guess it's most consistent to move it back to a `flatten` routine like the others vs. doing it a bit more tersely within here?

Reply:
Updated in new commit (though this will likely just get us back to the earlier issue).